data/Strings.lua

```diff
@@ -406,60 +406,60 @@ local Strings = {
         }
     },
     ["pl-pl"] = { -- Polish
-        thirst = "Thirst",
-        hunger = "Hunger",
-        fatigue = "Fatigue",
+        thirst = "Pragnienie",
+        hunger = "Głód",
+        fatigue = "Zmęczenie",
         step = {
             thirst = {
-                "You are hydrated, ",
-                "Low Dehydration, ",
-                "Dehydration, ",
-                "Severe Dehydration, ",
+                "Odpowiednie nawodnienie, ",
+                "Lekkie odwodnienie, ",
+                "Odwodnienie, ",
+                "Poważne odwodnienie, ",
             },
             hunger = {
-                "You are fed, ",
-                "Low Undernourishment, ",
-                "Undernourishment, ",
-                "Severe Undernourishment, ",
+                "Odpowiednie odżywienie, ",
+                "Lekkie niedożywienie, ",
+                "Niedożywienie, ",
+                "Poważne niedożywienie, ",
             },
             fatigue = {
-                "You are rested, ",
-                "Low Exhaustion, ",
-                "Exhaustion, ",
-                "Severe Exhaustion, ",
+                "Odpowiednie wypoczęcie, ",
+                "Lekkie zmęczenie, ",
+                "Zmęczenie, ",
+                "Poważne zmęczenie, ",
             }
         },
         settings = {
             category = {
-                game = "Game Settings",
-                display = "Display Settings"
+                game = "Ustawienia gry",
+                display = "Ustawienia wyświetlania"
```

Comment:
Both words are uppercased in english translation - I would go the same in all translation you've done so `Ustawienia Gry` and so on.

Reply:
Of course, all settings in CP2077 start with capital letter, but the next word (at least in the Polish version) starts with lower case (unlike English version, where all words are capitalized.). From what I can see, the Italian translation is made similar way. Once, I was translating another game and someone pointed out to me that in Polish, unlike English words, all next words should be in lower case.